src/lib/y2partitioner/actions/resize_blk_device.rb

```diff
@@ -76,6 +79,37 @@ def resize
         (result == :next) ? :finish : result
       end
 
+      # If the device seems to be empty the user has to confirm a warning stating
+      # the high risk of data loss.
+      #
+      # @return [Boolean] true iff there is no high risk or user accepted high risk
+      def confirm_empty
+        # If the device has descendants now resizing is ok even if
+        # nothing was probed on the device since the user already made
+        # an operation that overrides the device.
+        return true if @device.descendants.any?
+
+        # If the device does not exist in probed there cannot be an
+        # undetected file system or storage system on it.
+        return true if !@device.exists_in_probed?
```

Comment:
I would avoid to directly use the probed devicegraph when working with the Expert Partitioner. The Partitioner is designed to work with two given devicegraphs, one representing the system status (normally the probed one) and the second one is the working graph (equivalent to staging). So, I would modify the code a little bit, for example:

```
system_device = DeviceGraphs.instance.system.find_device(@device.sid)

return true if !system_device || system_device.descendants.any?

```